playbooks/cloud-provider-openstack-acceptance-test-flexvolume-cinder/run.yaml

```diff
@@ -95,25 +94,25 @@
           sleep 5
           # Run test
           export VOLUME_ID=$(cinder create 1 | awk '/ id / {print $4}')
-          "$kubectl" create -f examples/cinder-flexvolume/nginx.yaml
+          '{{ kubectl }}' create -f examples/cinder-flexvolume/nginx.yaml
 
           # If test passed
           if timeout 300 bash -c '
               while :
               do
-                  [[ $("$kubectl" describe pod nginx | awk "/^Status:/ {print \$2}") == Running ]] && break
+                  [[ $({{ kubectl }} describe pod nginx | awk "/^Status:/ {print \$2}") == Running ]] && break
```

Comment:
{{ kubectl }} should be '{{ kubectl }}'

Reply:
no $

Reply:
Stripped the single quotes is intentional, because the whole while loop has been quoted